src/resources/views/ui/inc/styles.blade.php

```diff
@@ -0,0 +1,37 @@
+@basset('https://unpkg.com/animate.css@4.1.1/animate.compat.css')
+@basset('https://unpkg.com/noty@3.2.0-beta-deprecated/lib/noty.css')
+@basset('https://cdn.jsdelivr.net/npm/prismjs@1.23.0/themes/prism.css')
+
+@basset('https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/css/line-awesome.min.css')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-regular-400.woff2')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-solid-900.woff2')
+@basset('https://cdnjs.cloudflare.com/ajax/libs/line-awesome/1.3.0/line-awesome/fonts/la-brands-400.woff2')
+
+@basset(base_path('vendor/backpack/crud/src/resources/assets/css/common.css'))
+
+<link rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap"
+      rel="stylesheet">
+{{-- TODO: We should use Basset for Google Fonts too, but it doesn't quite work yet: --}}
+{{-- @basset('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,400;0,600;0,700;1,400&display=swap',
+true, [], 'style') --}}
```

Comment:
Hmm this is a 🚩 - AFAIK `theme-coreuiv2` uses Source Sans Pro, but the other themes... they use something else... I think Tabler uses Inter. Are you sure this font is supposed to be here and not in the `theme_scripts`?